### PR TITLE
Personalize doc edits

### PR DESCRIPTION
**Pull request by shrouti1507**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/analytics/aws-personalize.md [hunk 1]
```diff
@@ -55,6 +55,14 @@ Please follow our [Adding a Source and Destination](https://docs.rudderstack.com
     * **Access Key ID**: The access key ID of your AWS account goes here.
     * **Secret Access Key**: Enter the secret access key of your AWS account.
     * **Region**: Please enter the region associated with your AWS account here.
+    * Also ensure that this actions are attached to the above access keys while setting up the aws policy for this keys. Below are the actions that you need to attach based on the type of events you want to send. If you are only sending putEvents, attach only `"personalize:PutEvents"`
```
Comment (RanjeetMishra) [inline destinations/analytics/aws-personalize.md:58]: nit: 
1. Also ensure that below actions are attached ...
2. setting up the aws policy for this keys ... this keys seems misplaced in statement?

**Review by RanjeetMishra — APPROVED** [review]
LGTM!
Usage of UT is kind of a blog document we need separately for Wh-Action and we can link here.
@ameypv-rudder

**Review by ruchiramoitra — COMMENTED** [review]
LGTM

File: destinations/analytics/aws-personalize.md (hunk 1 above)
Comment (gane5hvarma) [reply destinations/analytics/aws-personalize.md]: @ameypv-rudder can you beautify this statement. i wrote very badly

Comment (ameypv-rudder) [reply destinations/analytics/aws-personalize.md]: yes, sure.

Comment (ameypv-rudder) [reply destinations/analytics/aws-personalize.md]: Can you have a look once, Ganesh? I've reworked on the section to add clarity.

**Review by RanjeetMishra — APPROVED** [review]
LGTM!

**Pull request merged**